pyproject.toml

```diff
@@ -4,6 +4,10 @@ version = "2.1.0"
 description = "A collection of tools for interactive with GitHub"
```

Comment:
not a part of this pr, but... "for interactive" doesn't sound exactly right

Reply:
_I'm certain I don't know what you mean_